linode/manager
# upcoming: [M3-7701] - Add new ACLB Logo

## Description 📝

- Adds new ACLB Logo

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-01-24 at 5 05 25 PM](https://github.com/linode/manager/assets/115251059/cbb055a5-fed6-4c38-865d-504c9c3d0978) | ![Screenshot 2024-01-24 at 5 05 10 PM](https://github.com/linode/manager/assets/115251059/1599758b-4f0c-4bdc-a250-2fab2919ff90) |
| ![Screenshot 2024-01-24 at 5 10 03 PM](https://github.com/linode/manager/assets/115251059/5cc65fd5-0b80-4713-ac1a-5698e65545ce) | ![Screenshot 2024-01-24 at 5 08 58 PM](https://github.com/linode/manager/assets/115251059/6fd87c75-36fb-4bed-9cf7-77d0a7c6c166) |

## How to test 🧪

### Prerequisites
- Have ACLB feature flag enabled 

### Verification steps 
- Pull this PR or go to the preview link
- Verify you see the new ACLB logo in the primary navigation
- Verify you see the new ACLB logo in the Add New Menu
 
## As an Author I have considered 🤔
